buildlib/travis-build

```diff
@@ -10,6 +10,7 @@ cd build
 # The goal is warning free compile on latest gcc.
 CC=gcc-6 CFLAGS=-Werror cmake -GNinja ..
 ninja
+../buildlib/check-build --src ..
```

Comment:
Not sure I understand? The script has to be run at the right time and from the right cwd which won't trivially happen if it is put someplace else... Seems frail to move it.